addons/website/static/src/snippets/s_embed_code/options.js

```diff
@@ -0,0 +1,86 @@
+odoo.define('website.s_embed_code_options', function (require) {
+'use strict';
+
+const Dialog = require('web.Dialog');
+const core = require('web.core');
+const options = require('web_editor.snippets.options');
+
+const _t = core._t;
+
+options.registry.EmbedCode = options.Class.extend({
+    events: {
+        'click .o_we_edit_code': '_onEditClick',
```

Comment:
That is not how options are supposed to work, you bypass the whole editor system by doing that :wink: You should add a `data-` attribute on your button with the name of the option method to call, something like `data-edit-code=""` then declare a `editCode` method in your option here and all the events machinery would then be automatic.

Check how other options work for more example.